Task1/intro.json

```diff
@@ -1,238 +1,255 @@
-    "role": ["contributor", "maintainer"]
-  },
-  {
-     "name": "Lando Momo Paul",
-   "github_handle": "lando-paul",
-   "hacktoberfest_round": "first"
-   "role": ["contributor"]
-  },
-  {
-    "name": "Krishnapal Mori ",
-    "github_handle": "Krishnapalmori301",
-    "hacktoberfest_round": "first",
-    "role": ["contributor"]
-  },
-  {
-    "name": "Dhruv Gupta",
-    "github_handle": "MR-DHRUV",
-    "hacktoberfest_round": "first",
-    "role": ["contributor"]
-  },
-  {
-    "name": "Sonia Amehmbo",
-    "github_handle": "xennia316",
-    "hacktoberfest_round": "first",
-    "role": ["maintainer", "contributor"]
-  },
-  {
-    "name": "Aman Fangeria",
-    "github_handle": "amanfangeria980",
-    "hacktoberfest_round": "first",
-    "role": "contributor"
-  },
-  {
-    "name": "Dibyajit Tripathy",
-    "github_handle": "DibyajitTripathy",
-    "hacktoberfest_round": "third",
-    "role": ["contributor"]
-  },
-  {
-    "name": "Baimam Boukar Jean Jacques",
-    "github_handle": "baimamboukar",
-    "hacktoberfest_round": "second",
-    "role": ["contributor", "maintainer"]
-  },
-  {
-    "name": "Kshitij Verma",
-    "github_handle": "kshitijv256",
-    "hacktoberfest_round": "first",
-    "role": ["contributor"]
-  },
-  {
-    "name": "Nfon Andrew",
-    "github_handle": "andrew21-mch",
-    "hacktoberfest_round": "third",
-    "role": ["contributor"]
-  },
-  
-  {
-    "name": "Ravi",
-    "github_handle": "guptaravi540",
-    "hacktoberfest_round": "third",
-    "role": ["contributor"]
-  },
-  {
-    "name": "Hrishikesh Sunil Ghule",
-    "github_handle": "hrishikesh-g",
-    "hacktoberfest_round": "first",
-    "role": ["contributor"]
-  },
-  {
-    "name": "Vansh Goel",
-    "github_handle": "vansh-goel",
-    "hacktoberfest_round": "first",
-    "role": ["contributor"]
-  },
-  {
-    "name": "Kshitij kumar",
-    "github_handle": "Kshitij-10",
-    "hacktoberfest_round": "first",
-    "role": ["contributor"]
-  },
-  {
-    "name": "Agien Petra",
-    "github_handle": "petraAG",
-    "hacktoberfest_round": "second",
-    "role": ["contributor"]
-  },
-   {
-    "name": "Ayush Sarode",
-    "github_handle": "AyushSarode",
-    "hacktoberfest_round": "second",
-    "role": ["contributor"]
-  },
-  {
-    "name": "Joydeep Mallick",
-    "github_handle": "JoydeepMallick",
-    "hacktoberfest_round": "first",
-    "role": ["contributor"]
-   },
-  {
-  "name":"vishal singh",
-  "github_handle":"impetus007",
-  "hacktoberfest_round":"second",
-  "role":["contributor"]
-  },
-  {
-    "name": "fisher Monk",
-    "github_handle": "y)kw3ju5t3",
-    "hacktoberfest_round": "second",
-    "role": ["contributor"]
-  },
-  {
-    "name": "SudhanshuSharma",
-    "github_handle": "Sudhanshu-Sharma-22",
-    "hacktoberfest_round": "first",
-    "role": ["contributor", "maintainer"]
-
-  },
-  {
-    "name": "Azoh Munoh",
-    "github_handle": "azohMu",
-    "hacktoberfest_round": "first",
-    "role": ["contributor"]
-  },
-]

```

Comment:
[Check this out](https://github.com/Developer-Student-Clubs-UBa/hactoberfest2022/pull/159#issuecomment-1264360617)